packages/patternfly-4/react-core/src/components/Alert/Alert.md

```diff
@@ -174,3 +174,133 @@ class DangerAlert extends React.Component {
   }
 }
 ```
+
+## Static Live Region Alert
+```js
+import * as React from 'react';
+import { Alert } from '@patternfly/react-core';
+
+class StaticLiveRegionAlert extends React.Component {
+  render() {
+    return (
+      <React.Fragment>
+        <Alert toast={true} variant="info" title="Toast Alert Title"></Alert>
```

Comment:
do not have to specify boolean value `true` for toast. For variants, can we use AlertVariant?